### PR TITLE
PUB-740: Added e2e test for subscriptions button

### PR DESCRIPTION
**Pull request by johnoneill46**
Updated e2e tests to include "Subscriptions" header button that was missed in original (merged) PR of PUB-740

### CODE REVIEW

**Review by ChrisS1512 — APPROVED** [review]
Tests look good (Theres 1 lint error on the build that needs fixing)

File: src/main/views/template.njk [hunk 1]
```diff
@@ -18,12 +18,18 @@
             {
               href: "/otp-login-testing",
               text: "Subscriptions",
-              active: false
+              active: false,
+              attributes: {
```
Comment (commitBlob) [inline src/main/views/template.njk:22]: brilliant

**Review by commitBlob — APPROVED** [review]

File: src/test/e2e/Helpers/Selectors.ts [hunk 2]
```diff
@@ -3,6 +3,8 @@ module.exports = {
   ContinueButton: '.govuk-button',
   CommonPageTitle: '.govuk-heading-l',
   CommonPageTitleM: '.govuk-heading-m',
+  SubscriptionsButton: '#subscriptionHeaderButton',
+  SignInButton: "#signInButton",
```
Comment (commitBlob) [inline src/test/e2e/Helpers/Selectors.ts:7]: Build is complaining about these quotes, should be single

**Review by commitBlob — CHANGES_REQUESTED** [review]

File: src/test/e2e/Helpers/Selectors.ts (hunk 2 above)
Comment (johnoneill46) [reply src/test/e2e/Helpers/Selectors.ts]: Good spot - changes made accordingly

**Review by commitBlob — APPROVED** [review]

**Pull request merged**